modular_skyrat/modules/reagent_forging/code/forge.dm

```diff
@@ -462,6 +468,41 @@
 		COOLDOWN_START(spawned_glass, remaining_heat, glassblowing_amount)
 		return TRUE
 
+	if(istype(attacking_item, /obj/item/food))
+		var/obj/item/food/thing_to_cook = attacking_item
+		if(in_use)
+			to_chat(user, span_warning("You cannot do multiple things at the same time!"))
+			return
+		in_use = TRUE
+		if(forge_temperature < MIN_FORGE_TEMP)
+			fail_message(user, "The [src] is not hot enough to start cooking [thing_to_cook]!")
+			return
+		var/user_input = show_radial_menu(user, src, radial_options)
+		var/obj/item_to_spawn
+		if(!user_input)
+			fail_message(user, "No choice made")
+			return
+		balloon_alert_to_viewers("cooking...")
+		if(!do_after(user, 10 SECONDS, target = src))
+			fail_message(user, "You stop trying to cook [thing_to_cook]!")
+			return
+		switch(user_input)
+			if("oven")
+				var/datum/component/bakeable/item_bakeable_component = thing_to_cook.GetComponent(/datum/component/bakeable)
+				if(item_bakeable_component.bake_result)
+					item_to_spawn = item_bakeable_component.bake_result
+				else
+					item_to_spawn = /obj/item/food/badrecipe
```

Comment:
You could probably do this with a ternary operator instead (`var/my_var = a == 1 ? value_if_true : value_if_false`)

Reply:
these scare me